packages/react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -60,10 +60,15 @@ export const useStyles = css({
     isDisabled: {
       true: {
         cursor: 'not-allowed',
-        opacity: 0.58,
+        opacity: 0.38,
```

Comment:
ah yeah, I suggested a variable for this and now I'm convinced - disabledOpacity or similar

Reply:
Agreed. This isn't the only place we'll be using a disabled state like that. Plus, it would make it easier to choose a value that's not seemingly so arbitrary (`0.38` for now is fine because that's what's in the Figma currently)